src/core/Akka.Tests.Performance/Actor/Scheduler/DefaultSchedulerPerformanceTests.cs

```diff
@@ -0,0 +1,86 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="DefaultSchedulerPerformanceTests.cs" company="Akka.NET Project">
+//     Copyright (C) 2009-2016 Lightbend Inc. <http://www.lightbend.com>
+//     Copyright (C) 2013-2016 Akka.NET project <https://github.com/akkadotnet/akka.net>
+// </copyright>
+//-----------------------------------------------------------------------
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Akka.Actor;
+using NBench;
+
+namespace Akka.Tests.Performance.Actor.Scheduler
+{
+    /// <summary>
+    /// Designed to measure how many operations per-second the scheduler can perform in a fixed window of time.
+    /// 
+    /// Meant to measure the efficiency of the default <see cref="IScheduler"/> and to see if there's much contention
+    /// around the 
+    /// </summary>
+    public class DefaultSchedulerPerformanceTests
```

Comment:
Benchmark for the default `IScheduler` implementation. Not sure if this is the best design or not - wanted to test how much pressure could be put onto the work queue by multiple simultaneous queuers. There's a lot more that could be done here.